packages/cdk8s-cli/README.md

```diff
@@ -32,7 +34,9 @@ $ cd my-fun-little-project
 $ cdk8s init typescript-app
 ```
 
-## import - Import API objects as constructs
+## import
+
+### Import API objects as constructs
 
 To generate constructs for all Kubernetes API objects of a certain version, use
```

Comment:
This needs an update. `import` can now also be used to generate constructs for CRDs as well